notifications/notifiers/frontpage.py

```diff
@@ -88,7 +88,11 @@ def _posts_since_notification(notification_settings, notification):
         if not post.stickied and _is_post_after_notification(post, notification)
```

Comment:
Could this be added here?

```python
and not post._self_post.exclude_from_frontpage_emails
```

instead of filtering them out later below?

Reply:
No proxy_posts() replaces the reddit Submission object with an object that has ._self_post